frontend_vue/src/assets/fonts/typography.css

```diff
@@ -40,3 +40,12 @@
     url('./fa-solid-900.woff2') format('woff2'),
     url('./fa-solid-900.ttf') format('truetype');
 }
+
+@font-face {
```

Comment:
I think we're not planning to use this font anywhere else but on the CC token.
We might import it just on the CreditCard component

Reply:
ye could, don't think it buys us that much. leaving here for now